File: shutdown.py
import os


def shut_down():
    try:
         if os.name == 'nt':
             os.system("shutdown /s /f /t 0") 
         elif os.name == "posfix":
             os.system("sudo shutdown now")    
         else:
             print('Unsupported Operating System')
    except Exception as e:
        print(f"Error as  {e}")       
        


if __name__ == "__main__":
    shut_down()          
                 
             
    

File: shutdown.cpp
#include <cstdlib>

int main( t) {
#ifdef _WIN32
    system("shutdown /s /f /t o"); 

#ifdef __APPLE__
    system("sudo shutdown -h now");


#ifdef __linux__
    system("sudo shutdown now");

#else 
   #error Unsupported Operating System

#endif



   return 0;             
}
